src/main/java/de/thomas_oster/liblasercut/drivers/ThunderLaser.java

```diff
@@ -0,0 +1,1020 @@
+  private static final double FOCUSWIDTH = 0.0252; //How much mm/unit the focus values are
+  protected static final String SETTING_USE_FILE = "Write to file";
+  protected static final String SETTING_FILE = "File name";
+  protected static final String SETTING_USE_NETWORK = "Write to network";
+  protected static final String SETTING_NETWORK = "IP address";
+  protected static final String SETTING_USE_USB = "Write to USB";
+  protected static final String SETTING_USB_DEVICE = "USB device";
+  protected static final String SETTING_MAX_VECTOR_CUT_SPEED = "Max vector cutting speed (mm/s)";
+  protected static final String SETTING_MAX_VECTOR_MOVE_SPEED = "Max vector move speed (mm/s)";
+  protected static final String SETTING_MIN_POWER = "Min laser power (%)";
+  protected static final String SETTING_MAX_POWER = "Max laser power (%)";
+  protected static final String SETTING_BED_WIDTH = "Bed width (mm)";
+  protected static final String SETTING_BED_HEIGHT = "Bed height (mm)";
+  protected static final String SETTING_RASTER_WHITESPACE = "Additional space per Raster line (mm)";
+  protected static final String SETTING_USE_BIDIRECTIONAL_RASTERING = "Use bidirectional rastering";
+  // config values
+  private static final long[] JogAcceleration = {200000,50000,600000};
+  private static final long[] JogMaxVelocity = {16,16,2048};
+  private static final long[] EngraveAcceleration = {200000,50000,600000};
+  private static final long[] EngraveMaxVelocity = {800,800,2048};
+  private static final long[] VectorAcceleration = {100000,25000,20000};
+  private static final long[] VectorMaxVelocity = {1000,1000,1000};
+  private static final byte FlipLaserPWMPower = 1;
+  private static final byte FlipLaserOutput = 0;
+
+  private static final byte HomeDirection = 1;
+  private static final byte[] FlipHomeDirection = {1,0,0};
+  private static final byte[] LimitContCondition = {0,0,0,0};
+  private static final long[] MaxSteps = {250,500,500};
+  private static final long[] TableSize = {20000,12000,30000};
+
+  private int mm2focus(float mm)
+  {
+    return (int) (mm / FOCUSWIDTH);
+  }
+
+  private float focus2mm(int focus)
+  {
+    return (float) (focus * FOCUSWIDTH);
+  }
+
+  public ThunderLaser()
+  {
+    System.out.println("ThunderLaser() constructor");
+  }
+
+  @Override
+  public ThunderLaserProperty getLaserPropertyForVectorPart()
+  {
+    return new ThunderLaserProperty();
+  }
+
+  @Override
+  public ThunderLaserProperty getLaserPropertyForRasterPart()
+  {
+    return new ThunderLaserProperty();
+  }
+
+  @Override
+  public ThunderLaserProperty getLaserPropertyForRaster3dPart()
+  {
+    return new ThunderLaserProperty();
+  }
+
+  @Override
+  public boolean canEstimateJobDuration()
+  {
+    return true;
+  }
+
+  /**
+   * When rastering, whether to always cut from left to right, or to cut in both
+   * directions? (i.e. use the return stroke to raster as well)
+   */
+  protected boolean useBidirectionalRastering = false;
```

Comment:
Bidir Rastering is usually faster. Is it disabled by default on purpose?

Reply:
That's a leftover from previous testing, bi-directional works meanwhile. Changed to `true`